pc/localaudiosource.h

```diff
@@ -34,13 +34,14 @@ class LocalAudioSource : public Notifier<AudioSourceInterface> {
   void AddSink(AudioTrackSinkInterface* sink) override {}
   void RemoveSink(AudioTrackSinkInterface* sink) override {}
 
+  void Initialize(const cricket::AudioOptions* audio_options);
```

Comment:
Revert this change also and copy the initialise and local options to avoid changing webrtc codebase